tests/test_merge_db_functions.py

```diff
@@ -36,8 +36,8 @@
 from tuna.utils.merge_db import parse_jobline, parse_text_fdb_name, parse_text_pdb_name
 from tuna.utils.merge_db import best_solver, target_merge
 from tuna.utils.merge_db import update_master_list, write_merge_results
-from tuna.utils.merge_db import merge_text_file
-from tuna.utils.merge_db import get_sqlite_table
+from tuna.utils.merge_db import merge_text_file, merge_sqlite_pdb, merge_sqlite_bin_cache
+from tuna.utils.merge_db import merge_sqlite, get_file_list, get_sqlite_table
```

Comment:
Why add these functions, they don't appear to be in this file?

Reply:
Merging might have led these linking issues.